### PR TITLE
XRT-472 Porting XRT code to RHEL 8.1

### PR DESCRIPTION
**Pull request by rbramand-xilinx**
> Updated xrtdeps.sh to support rhel 8.1

### CODE REVIEW

File: src/runtime_src/tools/scripts/xrtdeps.sh [hunk 1]
```diff
@@ -276,15 +289,18 @@ prep_rhel()
     echo "Enabling EPEL repository..."
     rpm -q --quiet epel-release
     if [ $? != 0 ]; then
-	yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+	if [ $MAJOR -lt "8" ]; then
+	    yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+	else
+	    yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+	fi
```
Comment (stsoe) [inline src/runtime_src/tools/scripts/xrtdeps.sh:296]: Let's move this in prep_rhel7 and prep_rhel8.  Also move cmake3 install after prep_rhel8 and rhel7 ?

File: src/runtime_src/tools/scripts/xrtdeps.sh [hunk 2]
```diff
@@ -99,9 +98,17 @@ rh_package_list()
          systemd-devel \
          python3 \
          python3-pip \
-         systemd-devel \
```
Comment (stsoe) [inline src/runtime_src/tools/scripts/xrtdeps.sh:102]: Did you try building XRT from a clean centos8?  I am pretty sure this list is needed as it was.

Comment (chvamshi-xilinx) [reply src/runtime_src/tools/scripts/xrtdeps.sh:102]: @stsoe Looks like these are duplicates.

Comment (stsoe) [reply src/runtime_src/tools/scripts/xrtdeps.sh:102]: Duh :-)

File: src/runtime_src/tools/scripts/xrtdeps.sh (hunk 1 above)
Comment (rbramand-xilinx) [reply src/runtime_src/tools/scripts/xrtdeps.sh]: @stsoe made changes as suugested, please review

**Pull request merged**